R/dplyr_methods.R

```diff
@@ -892,23 +892,21 @@ group_split.SingleCellExperiment <- function(.data, ...) {
   
```

Comment:
the arguments shouldn't be `.tbl, ..., .keep = TRUE` https://dplyr.tidyverse.org/reference/group_split.html.

Also please have a look to the other functions roxygen, I'm pretty sure we do not redefine @param (?)

Reply:
Fixed :)